getting-started/scripts/install-tools

```diff
@@ -183,7 +188,7 @@ function main () {
   install_homebrew
   install_homebrew_casks
   install_homebrew_fomulae
-  rewrite_settings
+  #rewrite_settings
```

Comment:
Can you delete the `rewrite_settings` function?

Reply:
Sure thing, most recent commit should have removed the call to the function as well as the function itself.